Lib/test/test_bytes.py

```diff
@@ -1382,6 +1382,19 @@ def test_bytearray_api(self):
             except OSError:
                 pass
 
+    def test_mod_use_after_free(self):
+        # Prevent UAF in bytearray % (a1, a2) with re-entrant a[12].__repr__.
```

Comment:
"Re-entrant" is a wrong term. It is not re-entered here, and the issue not specific to `__repr__`. This is a concurrency issue, not re-entrancy issue.